docs/_posts/2019-07-31-531-release.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: blogpost
+title:  "5.3.1 Release Announcement"
+section_id: blog
+date:   2019-07-31 12:00:00
+author: Open GEE Development Team
+---
+
+<br />
+
+Hi Open GEE Community!
+ 
+We are excited to announce the official release of Open GEE 5.3.1!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+
+**Enhancements:**
+
+**- Uninstallation scripts are now copied to <code>/opt/google/install</code>.** This allow users to uninstall Fusion and Server without having to have a copy of the source code on hand.
+
+**- Memory and performance improvements for large builds.** Memory utilization and performance have been improved, especially for large builds.
+
+**- Increased logging during long Fusion operations.** Fusion can now be configured to log progress while calculating overlaps between resources.
```

Comment:
I think we should change "Fusion" to "System Manager"

Reply:
System Manager could be a too technical term. I changed it to "Fusion's system manager".